Core/src/org/sleuthkit/autopsy/communications/AccountsBrowser.java

```diff
@@ -65,20 +65,18 @@
 
     private final ExplorerManager accountsTableEM = new ExplorerManager();
     
-    final RelationshipBrowser relationshipBrowser;
+    final private RelationshipBrowser relationshipBrowser;
 
     /*
      * This lookup proxies the selection lookup of both he accounts table and
      * the messages table.
      */
     private final ProxyLookup proxyLookup;
 
-    public AccountsBrowser() {
+    public AccountsBrowser(RelationshipBrowser relationshipBrowser) {
```

Comment:
Just to repeat what codacy has said this Relationship browser argument and class variable can be removed as they are unused.